manager/assets/modext/util/utilities.js

```diff
@@ -548,12 +548,12 @@ MODx.util.url = {
         }
     },
     /**
-     * @property {Function} clearParams - Clears all dynamically set url parameters,
+     * @property {Function} clearAllParams - Clears all dynamically set url parameters,
      * while preserving those in a pre-defined list.
      *
      * @param {Object} stateData - Optional data to be used in subsequent url processing
      */
-    clearParams: function(stateData = {}) {
+    clearAllParams: function(stateData = {}) {
```

Comment:
Changes like this can be considered a breaking change, because it's a public APIs that's suddenly a different name. However this is one that was added in your previous PR, right? So that hasn't been released yet?

Reply:
Correct, that's new.